server/msgapi.py

```
@@ -70,26 +66,93 @@ def cors(environ):
     environ.headers['Access-Control-Allow-Headers']='*'
     return environ
 
+def build_session():
+    session = requests.Session()
+
+    # 开启 keep-alive（requests 默认是 keep-alive，但显式写更稳）
```

Ruff (0.14.3)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)